src/exceptiongroup/_exceptions.py

```diff
@@ -79,6 +80,20 @@ def message(self) -> str:
     def exceptions(self) -> tuple[EBase, ...]:
         return tuple(self._exceptions)
 
+    @property
```

Comment:
I am a bit conflicted on whether we should enforce py3.11 semantics on `__note__` when we cannot do so with any other exception class.

Reply:
I'm inclined to include this but I would love to hear your thoughts.

Reply:
I was (evidently) inclined to do so, but since we seem to have some weird interactions between `__dunder__` attributes and `@property` in CI I'm inclined to take it back out and just make this a standard attribute.